src/utils/transaction.ts

```diff
@@ -5,11 +5,29 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-
 import { Utxo } from '../wallet/types';
 import { UtxoError } from '../errors';
+import {crypto as cryptoBL, PrivateKey} from 'bitcore-lib'
 
 const transaction = {
+  /**
+   * Get the signature from the dataToSignHash for a private key
+   *
+   * @param {Buffer} dataToSignHash sha256 of a transaction's dataToSign.
+   * @param {PrivateKey} privateKey Signing key.
+   *
+   * @returns {Buffer}
+   *
+   * @memberof transaction
+   * @inner
+   */
+  sign(dataToSignHash: Buffer, privateKey: PrivateKey): Buffer {
```

Comment:
I think the same `getSignature` is better for this method, you are not signing anything. What do you think?

Reply:
Done.